### DISCUSSION TITLE
Remember position in feed after refresh

### DISCUSSION THREAD

**Discussion by doug8888** [j-fbriere/squawker#15] [Ideas]
Would it be possible for the app to remember my position in the feed after a refresh or after closing and re-opening Squawker?  This was possible in Twidere and makes it easy to continue reading my feed without having to scroll down to where I left off. Thanks for you efforts.

**Comment by j-fbriere** [comment]
Please create a [feature request](https://github.com/j-fbriere/squawker/issues/new?assignees=&labels=enhancement&projects=&template=feature_request.yml) instead of a making a comment in Discussions.
